### PR TITLE
Use i18next-browser-languagedetector (fixes #170)

### LINKED ISSUE #170: bug: I18n language reverts to default on refresh
**Describe the bug**
Every time I refresh the explorer with English/US as my default language, for about a half of a second the entire page renders in Spanish (catalan?). Then it re-renders in English.

**To Reproduce (please complete the following information)**
Refresh the page with user language set to anything other than the default

**System (please complete the following information):**
Firefox browser on MacOS
Browser default language set to English, location in US

### PR DESCRIPTION
**Pull request by NateWilliams2**

### CODE REVIEW

File: pages/_app.tsx [hunk 1]
```diff
@@ -36,6 +37,11 @@ const VocdoniApp: FC<NextAppProps> = ({ Component, pageProps }) => {
     console.log('these no has default providers');
   }
 
+  useEffect(() => {
+    setInitialRenderComplete(true);
+  }, []);
+
+  if (!initialRenderComplete) return <></>;
```
Comment (elboletaire) [inline pages/_app.tsx:44]: Why this change @NateWilliams2? 🤔

Comment (NateWilliams2) [reply pages/_app.tsx:44]: This change fixed a hydration error I was getting on loading the site:
```
Unhandled Runtime Error
Error: Hydration failed because the initial UI does not match what was rendered on the server.
```
I found this solution here: https://dev.to/ranjittherapids/troubleshooting-hydration-failed-error-in-nextjs-or-react-app-47fe

**Comment by elboletaire** [comment]
Super-seeded by the new explorer https://github.com/vocdoni/explorer
